lib/rpc/responses/send_transaction_response.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Soroban.RPC.Responses.SendTransactionResponse do
+  @moduledoc """
+  `SendTransactionResponse` struct definition.
+  """
+  @behaviour Soroban.RPC.Response.Spec
+
+  @type status :: :pending | :duplicate | :try_again_later | :error
```

Comment:
@Odraxs Since the response types will be built from maps, probably the status will be string instead of atom. So update to `"pending" | "duplicate" | ...`

Also update GetTransactionResponse module

Reply:
@miguelnietoa we can't set string literals in `@type` so I guess we just set it as `String.t()`, WDYT ?

Reply:
Agree!